frontend/packages/ux-editor/src/containers/FormDesigner.tsx

```diff
@@ -106,7 +111,11 @@ export const FormDesigner = ({ selectedLayout, selectedLayoutSet }: FormDesigner
             <LeftMenu className={classes.leftContent + ' ' + classes.item} />
             <FormContextProvider>
               <DesignView className={classes.mainContent + ' ' + classes.item} />
-              <RightMenu className={classes.rightContent + ' ' + classes.item} />
+              {/* key on rightMenu is used to force rerender when selectedLayout changes */}
```

Comment:
Since Prettier is now working as expected, I got a lot of formatting diffs here. But the actual code change is line 114 + 115. :)

Reply:
I've tested it locally and noticed that the selected component remains selected when switching between layouts:

https://github.com/Altinn/altinn-studio/assets/24462611/91dfd16f-29b3-47ab-bb54-9e5a3fa59098

It seems that the components can't be unselected after we merged this code:
https://github.com/Altinn/altinn-studio/pull/10900/commits/d6cb757a2493f3ec0eea341dceb31aad7cd0a4cc#diff-22a60e5e4d205167039c560ed733caab196465d697d94f6355b9f50e738d8c50R91

If we revert back the code, the right menu should rerender as expected as it uses the `formId` that gets cleared when switching layouts :
https://github.com/Altinn/altinn-studio/blob/68fc9bc8f64507e038f836375dd4d025f16a9161/frontend/packages/ux-editor/src/containers/FormContext.tsx#L110

Reply:
I created a PR to illustrate what I meant: https://github.com/Altinn/altinn-studio/pull/10945